client/src/ui/organisms/footer/index.scss

```diff
@@ -47,16 +47,35 @@
     ul {
       padding-inline-start: $base-spacing / 4;
     }
+  }
 
-    svg {
-      height: 18px;
-      vertical-align: sub;
-      width: 18px;
+  .social-icon::before {
+    background-size: 18px;
+    content: "";
+    display: inline-block;
+    filter: invert(1);
+    height: 18px;
+    vertical-align: sub;
+    width: 18px;
+  }
 
-      path {
-        fill: #fff;
-      }
-    }
+  .twitter::before {
+    background: transparent url("~@mdn/dinocons/social/twitter.svg") 0 0
+      no-repeat;
+  }
+
+  .github::before {
+    background: transparent url("~@mdn/dinocons/brands/github-mark-small.svg") 0
+      0 no-repeat;
+  }
+
+  .instagram::before {
+    background: transparent url("~@mdn/dinocons/social/instagram.svg") 0 0
+      no-repeat;
+  }
+
+  .visuall-hidden {
+    display: none;
   }
```

Comment:
I don't believe you will need to add this. The utility class should already be exposed by minimalist. On the relevant HTML element you should be able to add `className="visually-hidden"` and it should just work.